### ISSUE TITLE
Accessibility Scan

### ISSUE THREAD

**Issue by villanueval** [Smithsonian/Osprey#43]
The new accessibility scan (ticket RITM0090813) found four issues:

- [x] The tool buttons on the graph of the Summary tab of the homepage does not receive any keyboard focus. This still does not work for keyboard only users. The screenreaders read each of the graph tools as “graphic” or “link” without giving any context as to what those tools are 
- [x] The keyboard focus disappears after the “Summary” tab of the homepage until the footer. The keyboard focus should never disappear from sight 
- [x] For the graph on the Summary tab of the homepage, add a long description or a caption that gives a very brief overview such as the starting data point and the overall trend one sees when looking at a graph. Please add this before the table begins. See this page for more information [https://www.w3.org/WAI/tutorials/images/complex/] 
- [x] Hidden table “Cummulative number of images captured and objects digitized by the Digitization Program Office ...“ on https://dt-vmdpoqc01.si.edu/

**Comment by villanueval** [comment]
Graph is removed in 7b42ea470db912d010226c19355ea55542e5f3c5 and replaced with cards highlighting some projects.

**Comment by villanueval** [comment]
Issues identified by new scan:

## Level A

- [ ] Some pages have the same title , so the title cannot be used to distinguish pages. Change the title elements so they are unique for each page. [WCAG 2.1 A F25](https://www.w3.org/TR/WCAG20-TECHS/F25.html) [Section 508 (2017) A F25](https://www.w3.org/TR/WCAG20-TECHS/F25.html)
- [ ] The aria-controls attribute must point to IDs of elements in the same document. [WCAG 2.1 A 1.3.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation-programmatic) [Section 508 (2017) A 1.3.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation-programmatic) [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#aria-controls)

## Level AA

- [ ] Ensure that text and background colors have enough contrast [WCAG 2.1 AA 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) [Section 508 (2017) AA 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
  - [ ] Seems the "DEV VERSION" tag triggers this issue (`class="text-black bg-warning"`; [link to base template](https://github.com/Smithsonian/Osprey/blob/master/dashboard/web_app/templates/base.html#L113)
  - [ ] Team buttons on homepage using the white on light blue text (`"btn border-info btn-outline-info text-black"`)
- [ ] Headings should not be empty. Add text to the heading, or alt text if the heading contains an image. Screen readers read out page headings, allowing users to quickly skip to a section, but some older screen readers do not ignore empty headings. [WCAG 2.1 AA G130](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G130) [Section 508 (2017) AA G130](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G130)
  - [ ] Empty statistics in the Informatics Team page (`<div class="card-body"><h1></h1></div>...`)
- [ ] If you set any of the colors on the body or a elements you must set all of them. In HTML the color attributes are text , bgcolor , link , alink and vlink . In CSS the attributes are color and background-color . Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background. [WCAG 2.1 AA F24](https://www.w3.org/TR/WCAG20-TECHS/F24.html) [Section 508 (2017) AA F24](https://www.w3.org/TR/WCAG20-TECHS/F24.html) 
  - [ ] Seems to be in the SI footer. ([about page](https://github.com/Smithsonian/Osprey/blob/master/dashboard/web_app/templates/about.html#L19-L24))
  - [ ] Link to PM page in DPO ([PM link in dashboard](https://github.com/Smithsonian/Osprey/blob/master/dashboard/web_app/templates/dashboard.html#L60))
  - [ ] Link to projects in Team pages ([in query that builds the table contents](https://github.com/Smithsonian/Osprey/blob/master/dashboard/web_app/app.py#L573))

**Comment by villanueval** [comment]
Level A - Pagination in project dashboard returns the same title - Fixed in b1a18c9b719511b0792be47ec6034f7b59c5de3b

**Comment by villanueval** [comment]
Level A - The aria-controls attribute must point to IDs of elements in the same document. - Elements left behind from test navigation components. Removed.

Level AA - Ensure that text and background colors have enough contrast
	- One issue was a notice at the top of the pages. This was removed.
	- White on blue buttons were replaced with white-on-black or black-on-white buttons.
	
Level AA - Headings should not be empty. Add text to the heading, or alt text if the heading contains an image. 
	- Caused by a templated table with no data. Added code to not display if there are no rows in the table.

Level AA -  If you set any of the colors on the body or a elements you must set all of them. 
	- All elements were updated to specify text and background color together.

Level AA - The CSS outline or border style on this element makes it difficult or impossible to see the dotted link focus outline. 
	- Simplified styling of the form.

**Comment by villanueval** [comment]
There’s skipped headings on the homepage
 - Reformatted home to avoid these in 70c49402e1f497b181759f82aa52e086a5259c71

**Comment by villanueval** [comment]
New scan on 2023-09-22 just found false positives. We are good to go.
